src/stdlib/stdlib/std/base.tact

```diff
@@ -26,12 +84,28 @@ trait BaseTrait {
             let balanceBeforeMessage: Int = balance - ctx.value;
             if (balanceBeforeMessage < self.storageReserve) {
                 nativeReserve(self.storageReserve, ReserveExact);
-                send(SendParameters{bounce: bounce, to: to, value: 0, mode: SendRemainingBalance | SendIgnoreErrors, body: body, code: code, data: data });
+                send(SendParameters{
+                    bounce: bounce,
+                    to: to,
+                    value: 0,
+                    mode: SendRemainingBalance | SendIgnoreErrors,
+                    body: body,
+                    code: code,
+                    data: data,
```

Comment:
Let's use struct field punning here: `... bounce, to, value: 0, ...`

Reply:
Done